ui/raidboss/raidboss_config.ts

```diff
@@ -2174,13 +2174,17 @@ const templateOptions: OptionsTemplate = {
           'Nickname (Tini)': 'nick',
           'Role (Tank)': 'role',
           'Job (WAR)': 'job',
+          'Abbr Job (WAR)': 'jobAbbr',
+          'Full Job (Warrior)': 'jobFull',
           'Full Name (Tini Poutini)': 'name',
         },
         cn: {
-          '昵称 (Tini)': 'nick',
+          '昵称 (弗雷)': 'nick',
           '职能 (坦克)': 'role',
-          '职业 (战士)': 'job',
-          '全名 (Tini Poutini)': 'name',
+          '职业缩写 (DRK)': 'job',
+          '职业简称 (暗骑)': 'jobAbbr',
+          '职业全称 (暗黑骑士)': 'jobFull',
+          '全名 (弗雷)': 'name',
```

Comment:
This change makes `nick` and `name` looks like the same behaviour, also it looks like `黑骑` is the well-known abbreviation for Dark Knights much than `暗骑` that you wrote here in Chinese servers.

Reply:
I didn't modify the logic in this part; 'nick' and 'name' are the same in the Chinese server by default.

I use '暗'骑 instead of '黑'骑 because in PvP, the CN localization for DRK is translated as '暗,' while BLM is '黑.

Reply:
Okay, that sounds fair.